contributors/guide/pull-requests.md

```diff
@@ -55,7 +55,7 @@ You can run these local verifications before you submit your pull request to pre
 
 * Run and pass `make verify` (can take 30-40 minutes)
 * Run and pass `make test`
-* Run and pass `make test-integration`
```

Comment:
Why is this changing?

Reply:
Because the make target doesn't exist anymore: https://github.com/kubernetes/community/blob/master/Makefile#L25

Or do you imply that I should remove it?

Reply:
Oh, I understand the confusion. The contributor guide is hosted in this repo, but it isn't specific to this repo. That target does exist in the kubernetes core repo: https://github.com/kubernetes/kubernetes/blob/3a75a8c8d9e6a1ebd98d8572132e675d4980f184/build/root/Makefile#L193-L214

Reply:
Ah, ok. I'll take back the change for now then.